samples/bigquery_create_dataset.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2017, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+// [START bigquery_create_dataset]
+async function bigquery_create_dataset(DATASET_ID = 'YOUR_DATASET_ID') {
+  // Imports the Google Cloud client library
```

Comment:
Nit: In general we use imperatives as in, "you the developer should do this", as opposed to "this is what the code does."

This comment would become: "Import the Google Cloud client library"